.storybook/main.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  stories: ['../src/**/*.stories.@(js|tsx)'],
+  stories: ['./stories/**/*.stories.@(js|tsx)'],

```

Comment:
Why are we piggybacking on collocating stories with components? We've put an effort into it to have it well grouped and improve developer experience in that regard. Interested to hear your motivation here.

Reply:
Oh nvm, they weren't truly collocated, but in `src/stories` instead...

Reply:
yes, I had the exactly same thought process - moving them all to src was not was we intended. since storybook and the build need different configs sometimes, I ended up separating completely

Reply:
Do we still want to properly co-locate them? What is your view on it?